src/tests/mod.rs

```diff
@@ -15,17 +15,19 @@ mod literals;
 mod operators;
 
 fn eval_ok(nix_expr: &str) -> Value {
-    match evaluate(nix_expr) {
+    let workdir = std::env::current_dir().unwrap();
+    match evaluate(nix_expr, &workdir) {
         Ok(val) => val,
         Err(err) => panic!(
-            "eval '{}' shouldn't fail.\nError message: {}",
+            "eval '{}' shouldn't fail.\nError message: {:?}",
             nix_expr, err
         ),
     }
 }
 
 fn eval_err(nix_expr: &str) -> NixErrorKind {
-    evaluate(nix_expr)
+    let workdir = std::env::current_dir().unwrap();
+    evaluate(nix_expr, &workdir)
         .expect_err(&format!("eval '{}' expected an error", nix_expr))
```

Comment:
Same as above (feel free to ignore etc.):
```suggestion
        .expect_err(&format!("eval '{}' expected an error", nix_expr))
```
```suggestion
        .expect_err(&format!("eval '{nix_expr}' expected an error"))
```

Reply:
If you want these things to be picked up automatically, we can add clippy